pom.xml

```diff
@@ -1038,7 +1038,7 @@
             <dependency>
                 <groupId>org.apache.kafka</groupId>
                 <artifactId>kafka_2.12</artifactId>
-                <version>1.1.1</version>
+                <version>2.3.1</version>
```

Comment:
Could we extract this version as a property?

Reply:
Fixed.